2020/04/10 15:29:57 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /osmroads/14/13700/7518.json HTTP/1.1", upstream: "http://172.18.0.4:8088/osmroads/14/13700/7518.json", host: "192.168.178.4:1001"
2020/04/10 15:29:57 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:29:57 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /osmroads/14/13700/7518.json HTTP/1.1", upstream: "http://172.18.0.4:8088/osmroads/14/13700/7518.json", host: "192.168.178.4:1001"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /osmroads/14/13700/7518.json HTTP/1.1", upstream: "http://172.18.0.4:8088/osmroads/14/13700/7518.json", host: "192.168.178.4:1001"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /osmroads/14/13700/7518.json HTTP/1.1", upstream: "http://172.18.0.4:8088/osmroads/14/13700/7518.json", host: "192.168.178.4:1001"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /osmroads/14/13700/7518.json HTTP/1.1", upstream: "http://172.18.0.4:8088/osmroads/14/13700/7518.json", host: "192.168.178.4:1001"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /osmroads/14/13700/7518.json HTTP/1.1", upstream: "http://172.18.0.4:8088/osmroads/14/13700/7518.json", host: "192.168.178.4:1001"
2020/04/10 15:29:58 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:51:12 [error] 6#6: *2 upstream prematurely closed connection while reading upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/osmroads/14/13700/7518.json"
2020/04/10 15:57:00 [error] 6#6: *5 upstream prematurely closed connection while reading upstream, client: 192.168.178.3, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8088/favicon.ico", host: "192.168.178.4:1001", referrer: "http://192.168.178.4:1001/"
2020/04/20 17:56:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27404/15038.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27404/15038.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27401/15037.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27401/15037.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27401/15038.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27401/15038.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27401/15039.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27401/15039.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27401/15040.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27401/15040.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27402/15037.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27402/15037.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27402/15038.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27402/15038.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27402/15039.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27402/15039.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27402/15040.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27402/15040.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27403/15037.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27403/15037.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27403/15038.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27403/15038.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27403/15039.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27403/15039.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27403/15040.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27403/15040.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27404/15037.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27404/15037.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27404/15040.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27404/15040.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27404/15039.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27404/15039.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27405/15037.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27405/15037.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27405/15038.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27405/15038.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27405/15039.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27405/15039.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27405/15040.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27405/15040.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27406/15037.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27406/15037.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27406/15038.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27406/15038.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27406/15039.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27406/15039.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:18 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/15/27406/15040.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/15/27406/15040.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13700/7519.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13700/7519.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13700/7518.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13700/7518.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13701/7518.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13701/7518.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13700/7520.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13700/7520.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13701/7519.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13701/7519.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13701/7520.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13701/7520.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13702/7518.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13702/7518.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13702/7520.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13702/7520.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13702/7519.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13702/7519.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13703/7518.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13703/7518.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13703/7519.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13703/7519.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/04/20 17:56:22 [error] 6#6: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/14/13703/7520.pbf HTTP/1.1", upstream: "http://172.20.0.4:8088/api/tilestache/osmpoints/14/13703/7520.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4671 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/7/103/60.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/7/103/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4713 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/7/104/60.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/7/104/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4699 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/6/52/30.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/6/52/30.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4691 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/6/54/30.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/6/54/30.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4680 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/6/53/30.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/6/53/30.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4685 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/7/105/60.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/7/105/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4699 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/7/106/60.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/7/106/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4713 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/7/107/60.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/7/107/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4691 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/7/108/60.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/7/108/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4671 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/8/215/114.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/8/215/114.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4680 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/8/215/115.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/8/215/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4685 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/8/215/116.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/8/215/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4699 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/8/215/117.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/8/215/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4713 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/9/429/231.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/9/429/231.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4691 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/9/429/232.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/9/429/232.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4671 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/9/429/233.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/9/429/233.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4685 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/9/429/234.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/9/429/234.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4680 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/10/855/464.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/10/855/464.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4699 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/10/855/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/10/855/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4713 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/10/855/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/10/855/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/03 18:48:27 [error] 6#6: *4685 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/dataset_2_20200504003003/10/855/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/dataset_2_20200504003003/10/855/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/853/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/853/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/853/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/853/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/853/468.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/853/468.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/854/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/854/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/854/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/854/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/855/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/855/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/855/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/855/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/856/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/856/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/856/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/856/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/857/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/857/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/857/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/857/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/853/469.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/853/469.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/854/468.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/854/468.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/854/469.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/854/469.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/855/468.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/855/468.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/855/469.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/855/469.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/856/468.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/856/468.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/856/469.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/856/469.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/857/468.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/857/468.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/857/469.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/857/469.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/852/470.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/852/470.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/853/470.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/853/470.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/854/470.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/854/470.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/855/470.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/855/470.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/856/470.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/856/470.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/857/470.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/857/470.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/426/232.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/426/232.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/427/232.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/427/232.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/428/232.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/428/232.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/852/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/852/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/853/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/853/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/854/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/854/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/855/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/855/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/856/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/856/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/857/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/857/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/429/232.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/429/232.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/429/234.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/429/234.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/858/465.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/858/465.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/858/466.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/858/466.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/858/467.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/858/467.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/858/468.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/858/468.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/10/858/469.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/10/858/469.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/425/232.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/425/232.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/425/233.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/425/233.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/9/425/234.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/9/425/234.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/6/53/29.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/6/53/29.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/6/56/28.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/6/56/28.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/6/56/29.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/6/56/29.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/58.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/7/107/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:44:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/6/54/30.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/6/54/30.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/05 06:45:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/6/56/27.pbf HTTP/1.1", upstream: "http://172.22.0.7:8088/api/tilestache/osmpoints/6/56/27.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 09:42:02 [error] 6#6: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed?layer_name=fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed?layer_name=fsdfsd", host: "192.168.10.10:1001"
2020/05/06 09:42:51 [error] 6#6: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed?layer_name=fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed?layer_name=fsdfsd", host: "192.168.10.10:1001"
2020/05/06 09:42:52 [error] 6#6: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed?layer_name=fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed?layer_name=fsdfsd", host: "192.168.10.10:1001"
2020/05/06 09:44:51 [error] 6#6: *35 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/fsdfsd", host: "192.168.10.10:1001"
2020/05/06 12:50:12 [error] 6#6: *1205 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/58.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 12:52:23 [error] 6#6: *1206 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 12:52:47 [error] 6#6: *1204 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 12:53:08 [error] 6#6: *1208 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 12:53:45 [error] 6#6: *1209 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/211/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/211/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 12:58:54 [error] 6#6: *1202 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/212/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/212/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:10:16 [error] 6#6: *1205 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/213/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/213/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:12:27 [error] 6#6: *1206 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:12:54 [error] 6#6: *1204 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:13:13 [error] 6#6: *1208 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/216/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/216/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:13:51 [error] 6#6: *1209 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/105/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/105/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:18:55 [error] 6#6: *1202 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/106/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/106/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/209/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/209/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/58.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/108/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/108/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/58.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/209/116.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/209/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/105/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/105/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/104/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/104/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/209/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/209/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/106/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/106/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/208/114.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/208/114.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/208/116.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/208/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/208/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/208/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/208/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/208/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/208/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/208/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/209/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/209/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/211/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/211/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/210/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/210/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/212/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/212/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/213/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/213/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/207/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/207/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/207/116.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/207/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/207/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/207/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/103/57.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/103/57.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/207/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/207/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/103/58.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/103/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/103/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/103/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/207/114.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/207/114.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/208/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/208/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/209/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/209/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/207/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/207/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/210/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/210/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/211/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/211/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/212/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/212/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/209/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/209/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/211/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/211/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/212/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/212/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/210/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/210/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/213/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/213/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/213/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/213/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/104/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/104/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/105/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/105/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/106/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/106/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/58.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/116.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/116.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:30 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/120.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/120.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/108/58.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/108/58.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/216/116.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/216/116.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/108/60.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/108/60.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/108/59.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/108/59.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/216/117.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/216/117.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/216/118.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/216/118.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/216/119.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/216/119.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/105/57.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/105/57.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/106/57.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/106/57.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/107/57.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/107/57.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/7/108/57.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/7/108/57.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1607 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/211/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/211/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1204 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/212/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/212/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1205 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/214/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/214/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1209 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/213/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/213/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1202 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/216/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/216/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 13:42:31 [error] 6#6: *1476 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/8/215/115.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/8/215/115.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 17:05:31 [error] 6#6: *1689 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /api/tilestache/osmpoints/6/53/29.pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/api/tilestache/osmpoints/6/53/29.pbf", host: "192.168.10.10:1001", referrer: "http://localhost:3000/"
2020/05/06 17:07:55 [error] 6#6: *1719 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/fsdfsd", host: "192.168.10.10:1001"
2020/05/06 17:08:38 [error] 6#6: *1719 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/fsdfsd", host: "192.168.10.10:1001"
2020/05/06 17:09:25 [error] 6#6: *1719 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/fsdfsd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/fsdfsd", host: "192.168.10.10:1001"
2020/05/07 05:18:37 [error] 6#6: *1820 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints", host: "192.168.10.10:1001"
2020/05/07 06:30:21 [error] 6#6: *1847 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints!gfd HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints!gfd", host: "192.168.10.10:1001"
2020/05/07 07:41:36 [error] 6#6: *1883 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf", host: "192.168.10.10:1001"
2020/05/07 07:58:36 [error] 6#6: *1908 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf", host: "192.168.10.10:1001"
2020/05/07 09:50:44 [error] 6#6: *1944 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf", host: "192.168.10.10:1001"
2020/05/07 09:52:39 [error] 6#6: *1947 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 09:54:08 [error] 6#6: *1947 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 09:55:51 [error] 6#6: *1947 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 09:56:39 [error] 6#6: *1947 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:01:27 [error] 6#6: *1956 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:03:22 [error] 6#6: *1956 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:07:00 [error] 6#6: *1956 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:07:50 [error] 6#6: *1956 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:10:09 [error] 6#6: *1956 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:13:54 [error] 6#6: *1969 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:18:25 [error] 6#6: *1972 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:18:42 [error] 6#6: *1972 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:20:37 [error] 6#6: *1972 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:23:06 [error] 6#6: *1982 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:25:40 [error] 6#6: *1982 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:25:53 [error] 6#6: *1982 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:26:37 [error] 6#6: *1982 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:26:49 [error] 6#6: *1982 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=1%202%203%204%205&extension=pbf&verbose=True", host: "192.168.10.10:1001"
2020/05/07 10:28:47 [error] 6#6: *1994 upstream prematurely closed connection while reading response header from upstream, client: 192.168.10.8, server: localhost, request: "GET /tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=12&extension=pbf&verbose=True HTTP/1.1", upstream: "http://172.22.0.2:8088/tilestache/seed/osmpoints?bbox=22.024546%20144.711914%201.999106%20102.897949&zooms=12&extension=pbf&verbose=True", host: "192.168.10.10:1001"
